File: Phasegate/DayOne/Level 1/java/TEST1/TestTaskOne.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;
public class TestTaskOne {
    @Test
    public void testEvenNumbers() {
        int[] number   = {45, 60, 3, 10, 9, 22};
        int[] expected = {60, 10, 22};
        int[] actual   = TaskOne.evenNumbers(number);
        assertArrayEquals(expected, actual);
    }
    @Test
    public void testOddNumbers() {
        int[] number   = {45, 60, 3, 10, 9, 22};
        int[] expected = {45, 3, 9};
        int[] actual   = TaskOne.oddNumbers(number);
        assertArrayEquals(expected, actual);
    }
}

File: Phasegate/DayOne/Level 1/python/TEST/test_task_one.py
from unittest import TestCase
import task_one
class task_one_test(TestCase):
    def test_that_even_numbers_are_returned(self):
        numbers  = [45, 60, 3, 10, 9, 22]
        actual   = task_one.even_numbers(numbers)
        expected = [60, 10, 22]
        self.assertEqual(actual, expected)
    def test_that_odd_numbers_are_returned(self):
        numbers  = [45, 60, 3, 10, 9, 22]
        actual   = task_one.odd_numbers(numbers)
        expected = [45, 3, 9]
        self.assertEqual(actual, expected)
